### ISSUE TITLE
rb-readline error when scrolling down on stories listed in history

### ISSUE THREAD

**Issue by kylegearhart** [story-branch/story_branch#26]

**Comment by kylegearhart** [comment]
```
Connecting with Pivotal Tracker
Getting stories...
XXXXXXXX - Pay API Details
XXXXXXXX - 6702: changing PIN.
XXXXXXXX - while return back to change username page from previous page
Leave blank to exit, use <up>/<down> to scroll through stories, TAB to list all and auto-complete
Select a story: XXXXXXXX - Details/Users/xxxxxxxx/.rvm/gems/ruby-2.2.1/gems/rb-readline-0.5.4/lib/rbreadline.rb:3494:in `[]=': incompatible character encodings: ASCII-8BIT and UTF-8 (Encoding::CompatibilityError)
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/rb-readline-0.5.4/lib/rbreadline.rb:3494:in `rl_redisplay'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/rb-readline-0.5.4/lib/rbreadline.rb:4728:in `_rl_internal_char_cleanup'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/rb-readline-0.5.4/lib/rbreadline.rb:4789:in `readline_internal_charloop'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/rb-readline-0.5.4/lib/rbreadline.rb:4853:in `readline_internal'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/rb-readline-0.5.4/lib/rbreadline.rb:4875:in `readline'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/rb-readline-0.5.4/lib/readline.rb:45:in `readline'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/story_branch-0.2.11/lib/story_branch.rb:477:in `readline'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/story_branch-0.2.11/lib/story_branch.rb:405:in `select_story'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/story_branch-0.2.11/lib/story_branch.rb:126:in `create_story_branch'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/gems/story_branch-0.2.11/bin/git-story:4:in `<top (required)>'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/bin/git-story:23:in `load'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/bin/git-story:23:in `<main>'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/pivotal/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by rpbaltazar** [comment]
In which scenario does this happen? does by any chance your story have a non supported character?

**Comment by jasonm23** [comment]
It's possible this tracker project had Unicode/emoji in the names.  

BTW tracker now supports emojis etc. in story data.

**Comment by rpbaltazar** [comment]
This should be fixed with #51

**Comment by rpbaltazar** [comment]
If the issue was with emojis then this is solved with #51
